collections/_article/dev-snapshot-godot-4-5-dev-3.md

```diff
@@ -0,0 +1,111 @@
+date: 2025-04-25 12:00:00
+---
+
+If the past snapshot was any indication, you might think that development updates aren't slowing down anytime soon. And… You'd be right! Indeed, progress has been booming, and the amount of new features added even compared to the previous release is staggering; curating this selection was especially difficult. As always, new features mean new bugs in need of fixing, so we encourage everyone interested to get feedback and bug reports in as easrly as possible.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.5.dev3/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+---
+
+*The cover illustration is from* [**Cornerpond**](https://store.steampowered.com/app/3454590/Cornerpond/?curator_clanid=41324400), *a fishing game that takes place entirely in the corner of your desktop. It is developed by [foolsroom](https://foolsroom.itch.io/). You can wishlist the game [on Steam](https://store.steampowered.com/app/3454590/Cornerpond/?curator_clanid=41324400) and follow the developer on [Twitter](https://twitter.com/foolsroom).*
+
+## Highlights
+
+In case you missed them, see the [4.5 dev 1](/article/dev-snapshot-godot-4-5-dev-1/) and [4.5 dev 2](/article/dev-snapshot-godot-4-5-dev-2/) release notes for an overview of some key features which were already in that snapshot, and are therefore still available for testing in dev 3.
+
+### Screen reader support
+
+Accessability should be every developer's top priority, full-stop. Someone being excluded from an experience for factors outside of their control is an area that videogames and applications have the potential to circumvent entirely. It does, however, take a solid framework to allow such accomodations to take place. To streamline this process for everyone—players and developers alike—our resident tech guru [bruvzg](https://github.com/bruvzg) took to the absolutely Herculean task of integrating [AccessKit](https://github.com/AccessKit/accesskit) to Godot as a whole.
+
+[GH-76829](https://github.com/godotengine/godot/pull/76829) was a project two years in the making, changing over **32,000** lines of code, and incurred hundreds of comments with feedback/testing. Users are encouraged to look at the pull request for more information, as there's no feasible way we could properly summarize these changes. Unsurprisingly, this was *by far* the change with the most ramifications of the entire snapshot, so much so that it's already seen [multiple](https://github.com/godotengine/godot/pull/105197) [fixes](https://github.com/godotengine/godot/pull/105216) caused by its regressions, but it's well worth it. After all, accessability is our top priority!
```

Comment:
```suggestion
[GH-76829](https://github.com/godotengine/godot/pull/76829) was a project two years in the making, changing over **32,000** lines of code, and incurred hundreds of comments with feedback/testing. Users are encouraged to look at the pull request for more information, as there's no feasible way we could properly summarize these changes. Unsurprisingly, this was *by far* the change with the most ramifications of the entire snapshot, so much so that it's already seen [multiple](https://github.com/godotengine/godot/pull/105197) [fixes](https://github.com/godotengine/godot/pull/105216) caused by its regressions, but it's well worth it. After all, accessibility is our top priority!
```